2025-03-19 15:39:30,485 ERROR [/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/utils/log.py] Internal Server Error: / (<class 'django.db.utils.ProgrammingError'>, ProgrammingError('column post_posts.preview_en does not exist\nLINE 1: ..., "post_posts"."body_ru", "post_posts"."preview", "post_post...\n                                                             ^\nHINT:  Perhaps you meant to reference the column "post_posts.preview".\n'), <traceback object at 0x7f306c9a35c0>)
Traceback (most recent call last):
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedColumn: column post_posts.preview_en does not exist
LINE 1: ..., "post_posts"."body_ru", "post_posts"."preview", "post_post...
                                                             ^
HINT:  Perhaps you meant to reference the column "post_posts.preview".


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 199, in render
    len_values = len(values)
                 ^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/query.py", line 382, in __len__
    self._fetch_all()
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: column post_posts.preview_en does not exist
LINE 1: ..., "post_posts"."body_ru", "post_posts"."preview", "post_post...
                                                             ^
HINT:  Perhaps you meant to reference the column "post_posts.preview".

2025-03-19 15:39:30,487 ERROR [/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/core/servers/basehttp.py] "GET / HTTP/1.1" 500 211159 None
2025-03-19 15:42:02,920 ERROR [/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/utils/log.py] Internal Server Error: / (<class 'django.db.utils.ProgrammingError'>, ProgrammingError('column post_posts.preview_en does not exist\nLINE 1: ..., "post_posts"."body_ru", "post_posts"."preview", "post_post...\n                                                             ^\nHINT:  Perhaps you meant to reference the column "post_posts.preview".\n'), <traceback object at 0x7d19c166b240>)
Traceback (most recent call last):
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedColumn: column post_posts.preview_en does not exist
LINE 1: ..., "post_posts"."body_ru", "post_posts"."preview", "post_post...
                                                             ^
HINT:  Perhaps you meant to reference the column "post_posts.preview".


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 199, in render
    len_values = len(values)
                 ^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/query.py", line 382, in __len__
    self._fetch_all()
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: column post_posts.preview_en does not exist
LINE 1: ..., "post_posts"."body_ru", "post_posts"."preview", "post_post...
                                                             ^
HINT:  Perhaps you meant to reference the column "post_posts.preview".

2025-03-19 15:42:02,923 ERROR [/home/pelagusasrock/PycharmProjects/DjangoProject7-2-flush/.venv/.venv/lib/python3.12/site-packages/django/core/servers/basehttp.py] "GET / HTTP/1.1" 500 211001 None
